src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+import nlpaug.augmenter.char as nac
+
+
+def keyboard_typo(
+    texts: List[str],
+    **kwargs,
+) -> List[str]:
+    '''Generate keyboard typo perturbed texts for augmentation.
+
+    Args:
+        texts: List of texts to be augmented.
```

Comment:
I think showing some arguments of `nlpaug.augmenter.char.keyboard.KeyboardAug` explicitly here is helpful for langcheck users. At least

- aug_char_p (float)
- aug_char_min (int)
- aug_char_max (int)
- aug_word_p (float)
- aug_word_min (int)
- aug_word_max (int)

seem general enough to add to the list of langcheck arguments. WDYT?